tendrl/node_agent/ansible_runner/ansible_module_runner.py

```diff
@@ -2,10 +2,10 @@
 from ansible import modules
 import os
 import subprocess
-from tendrl.node_agent.config import TendrlConfig
+from tendrl.common.config import TendrlConfig
 import uuid
 
-config = TendrlConfig()
+config = TendrlConfig("node_agent", "/etc/tendrl/tendrl.conf")
```

Comment:
The python project name "node_agent" and the config file path should not be hardcoded